.github/workflows/release-part-2.yml

```diff
@@ -76,7 +76,7 @@ jobs:
                           Workbench-macOS-binary \
                           InVEST-sample-data \
                           InVEST-user-guide \
-                          "Source distribution"
+                          "Source distribution macos-13 3.12"
```

Comment:
So we're just choosing an arbitrary source distribution to check, but choosing not to check both Windows and Mac source distributions? Just want to make sure I understand and then wondering if we should have a note about this being hard coded and that eventually we'll need to update this manually.

Reply:
I'm not 100% comfortable with hardcoding this, but yes, we need to pick one of the source distributions and python versions to upload here.  We're not checking anything, though, just downloading one of the `.tar.gz` artifacts and uploading it to the github release object.

Having said that, I dislike having a statically-defined string here that will need to be updated in the future, so I've fixed it in e5a3065e6